src/custom/constants/tokens/index.ts

```diff
@@ -145,11 +171,13 @@ export const ADDRESS_IMAGE_OVERRIDE = {
 export const MERKLE_DROP_CONTRACT_ADDRESSES: Record<number, string> = {
   [SupportedChainId.MAINNET]: '0x64646f112FfD6F1B7533359CFaAF7998F23C8c40',
   [SupportedChainId.RINKEBY]: '0x5444c4AFb2ec7f7367C10F7732b8558650c5899F',
+  [SupportedChainId.GOERLI]: '0x5444c4AFb2ec7f7367C10F7732b8558650c5899F', // TODO: update with actual goerli addresses
```

Comment:
This is something I'm not sure will make sense.
Will we replicate the airdrop on goerli? 🤔 

Need to clarify that with the backend

Reply:
Mentioned in https://github.com/cowprotocol/contracts/issues/11 issue